### ISSUE TITLE
toBuilder() method not recognised by Intellij

### ISSUE THREAD

**Issue by enr74** [mplushnikov/lombok-intellij-plugin#152]
According to Lombok's documentation:
> you can use @Builder(toBuilder = true) to also generate an instance method in your class called  toBuilder(); it creates a new builder that starts out with all the values of this instance.  

but Intellij / lombok plugin doesn't recognise the toBuilder() method, even it works at runtime. 
```
import lombok.Builder;
import lombok.Getter;

@Builder(builderClassName="Builder", toBuilder=true)
@Getter
public class Example {
    private String text;
}
```
The unit test works fine, but the toBuilder() method is highlighted in red:
```
public class ExampleTest {
    @Test
    public void test() {
        Example example = Example.builder().text("test").build();
        Example clone = example.toBuilder().build();
        assertEquals(example.getText(), clone.getText());
        assertNotNull(clone);
    }
}
```

I'm using  Intellij IDEA 15 CE, JDK 1.8.0_11, Lombok Plugin v. 0.9.6.14

**Comment by tomaszganski** [comment]
it's duplication of #134, which will be resolved in next release

**Comment by mplushnikov** [comment]
Closed as duplicate of #134 
Thank you Tomek!
